cmd/dhcpserver/get.go

```diff
@@ -47,12 +47,18 @@ var getCmd = &cobra.Command{
 			return err
 		}
 
-		servers, err := pvmclient.DHCPClient.Get(id)
+		server, err := pvmclient.DHCPClient.Get(id)
 		if err != nil {
 			return fmt.Errorf("failed to get a dhcpserver, err: %v", err)
 		}
-
-		spew.Dump(servers)
+		var IPandMAC string
+		for _, lease := range server.Leases {
+			IPandMAC += fmt.Sprintf("%s - %s ", *lease.InstanceIP, *lease.InstanceMacAddress)
+		}
+		table := utils.NewTable()
+		table.SetHeader([]string{"Network Name", "IP - MAC", "Status"})
+		table.Append([]string{*server.Network.Name, IPandMAC, *server.Status})
```

Comment:
Even here the network name can be nil when server status is BUILDING.

Reply:
Agree, made changes to return a log to retry later.